Celeste.Mod.mm/Mod/Everest/Everest.DebugRC.cs

```diff
@@ -374,7 +374,7 @@ public static void Write(HttpListenerContext c, string str) {
                     Handle = c => {
                         NameValueCollection data = ParseQueryString(c.Request.RawUrl);
 
-                        string sid = data["area"];
+                        string sid =  WebUtility.UrlDecode(data["area"]);
```

Comment:
nitpick: double space could be a single space
```suggestion
                        string sid = WebUtility.UrlDecode(data["area"]);
```